src/Host/Host.php

```diff
@@ -202,31 +192,33 @@ public function multiplexing(bool $multiplexing = true)
         return $this;
     }
 
-    /**
-     * @return array
-     */
-    public function getOptions()
+    public function getSshArguments() : Arguments
     {
-        return $this->options;
+        $this->initOptions();
+        return $this->sshArguments;
     }
 
-    /**
-     * @param array $options
-     * @return $this
-     */
-    public function options(array $options)
+    public function sshOptions(array $options) : Host
     {
-        $this->options = $options;
+        $this->sshArguments = $this->sshArguments->withOptions($options);
         return $this;
     }
 
-    /**
-     * @param string $option
-     * @return $this
-     */
-    public function addOption(string $option)
+    public function sshFlags(array $flags) : Host
+    {
+        $this->sshArguments = $this->sshArguments->withFlags($flags);
+        return $this;
+    }
+
+    public function addSshOption(string $option, $value) : Host
+    {
+        $this->sshArguments = $this->sshArguments->withOption($option, $value);
```

Comment:
I don't like this approach (btw i like immutability). Why dot just call `$this->sshArguments->addOption($option, $value);`?